ops/ansible/roles/bfd-server-load/test/run-tests.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# Stop immediately if any command returns a non-zero result.
+set -eou pipefail
+
+function help() {
+    echo "This script runs our test cases locally, via Docker."
```

Comment:
Is this script only for running the load tests locally, or is this a helper for testing the load test's architecture?

Reply:
This script is used only for testing the Ansible playbook(s) for this `server-load` service, and was copied from the same testing scripts from other Ansible roles. So, yep, it's just a helper for testing the Ansible infrastructure-as-code for the load tests.